### ISSUE TITLE
Add parameter for learning_rate schedule

### ISSUE THREAD

**Issue by alexisrozhkov** [alexisrozhkov/rele#4]
Currently training for a long time seems to make dqn agent "forget" good policy and "overwrite" it with some suboptimal, probably due to some "outliers" in replay memory, which are poorly represented by network, and which "pull" its updates in some arbitrary direction. Gradual decay of learning rate should address this issue (if tuned properly)
